extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -1660,6 +1705,25 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				data.push([
 					sourceDatabaseName,
 					targetDatabaseInfo?.databaseName,
+					// If key not added to _schemaMigrationHasCompleted then schema migration hasn't started.
+					<azdata.HyperlinkColumnCellValue>{				// Migrate schema
+						icon: schemaMigrationStatus === undefined
+							? IconPathHelper.notStartedMigration
+							: schemaMigrationStatus === SchemaMigrationState.CompletedWithError
```

Comment:
move IconpathHelper to util, i.e. https://github.com/microsoft/azuredatastudio/blob/main/extensions/sql-migration/src/api/utils.ts#L359